code/modules/mob/living/carbon/life.dm

```diff
@@ -372,9 +383,10 @@
 					healths.icon_state = "health6"
 		else
 			healths.icon_state = "health7"
-	handle_hud_icons_health_overlay()
 
-/mob/living/carbon/proc/handle_hud_icons_health_overlay()
+/mob/living/carbon/update_damage_hud()
+	if(!client)
```

Comment:
eventually `healths` needs to be moved from being a mob side screen element to a UI hud element...